userapi/storage/accounts/sqlite3/deltas/20200929203058_is_active.sql

```diff
@@ -0,0 +1,38 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE account_accounts RENAME TO account_accounts_tmp;
```

Comment:
I'm curious about the choice to recreate the table in this way - can we not add columns in SQLite using `ALTER TABLE ADD`?

Reply:
I don't think sqlite supports "IF NOT EXISTS" as postgres does.

The rationale here is that if someone attempts to run this migration, it will work even if the database is already up to date (but goose migration table does not exist). However, all accounts will be reactivated in the process.

Also, I've not tried, but for some column types it is not possible to add a default values with ALTER.